src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import style from './Tabs.st.css';
+import classNames from 'classnames';
+import { CONTENT_ALIGNMENT, SKIN, CONTENT_WIDTH } from './constants';
+
+export interface TabsProps {
+  items: TabItem[];
+  onClick?: Function;
```

Comment:
call it `onTabClick()`, to indicate this is not just a regular container click.
also - define its type so it will receive a parameter from type `TabItem`